src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -83,6 +83,8 @@
             "alignments as well as the proportion of the reads that passed machine signal-to-noise threshold quality filters. " +
             "Note that these quality filters are specific to Illumina data; for additional information, please see the corresponding " +
             "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6329'>GATK Dictionary entry</a>. </p>" +
+            "" +
+            "Note: For all Picard tools, headers that refer to PCT in the output metrics files, are referring to a fraction, not percentage." +
```

Comment:
I think a more explicit way to put this would be:

All metrics labeled as percentages (with "percent" in the full metric name or "PCT" in the name given in the output file) are actually expressed as fractions. For example, "PCT_TARGET_BASES_20X = 0.85" should be interpreted as "85 percent of target bases are covered to 20X coverage or more".